extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -563,7 +563,13 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
 
             if (_self._config.enableAjaxErrorStatusText && xhr.status >= 400) {
                 dependency.properties = dependency.properties || {};
-                dependency.properties.responseText = xhr.statusText + " - " + xhr.responseText;
+                const responseType = xhr.responseType;
+                if (responseType === '' || responseType === 'text') {
```

Comment:
Will responseType ever be undefined/null? Does empty string always mean text?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType

Reply:
@markwolff talked to Nev and double checked with doc, responseType by default is "", and empty string always mean text.